README.md

```diff
@@ -16,7 +16,7 @@ Add accepts parser to fastify
 const fastify = require('fastify')
 const Boom = require('boom')
 
-fastify.register(require('@fastify/accepts'))
+await fastify.register(require('@fastify/accepts'))
```

Comment:
I don't think it required `await` here.

Reply:
Isn't route registration sync now, so it needs the `await`?

https://www.fastify.io/docs/latest/Guides/Migration-Guide-V4/#synchronous-route-definitions

Reply:
The logic applied to the plugin that use `onRoute` hooks.
Since the route registration is `async`, if you do not wait the plugin to finish it's registration.
The route will be missed in that `onRoute` hooks.

This do not apply to this plugin since it only modify the `request` and `reply` decorators.

Reply:
Thanks @climba03003, had reread the docs and noticed it was for just `onRoute` hooks a second ago.